subaru_outback_2015_eyesight.dbc

```diff
@@ -163,26 +162,30 @@ BO_ 352 ES_Brake: 8 XXX
  SG_ Brake_On : 22|1@0+ (1,0) [0|3] "" XXX
  SG_ Brake_Light : 20|1@0+ (1,0) [0|2047] "" XXX
  SG_ Cruise_Activated : 23|1@0+ (1,0) [0|3] "" XXX
- SG_ Brake_Pressure : 0|16@1+ (1,0) [0|255] "" XXX
  SG_ ES_Error : 21|1@0+ (1,0) [0|7] "" XXX
+ SG_ Brake_Pressure : 0|16@1+ (1,0) [0|255] "" XXX
 
 BO_ 353 ES_CruiseThrottle: 8 XXX
+ SG_ SET_2 : 12|4@1+ (1,0) [0|15] "" XXX
+ SG_ Checksum : 56|8@1+ (1,0) [0|15] "" XXX
+ SG_ Button : 48|3@1+ (1,0) [0|3] "" XXX
+ SG_ ES_Error : 42|1@1+ (1,0) [0|3] "" XXX
+ SG_ Standstill_2 : 41|1@1+ (1,0) [0|3] "" XXX
+ SG_ Standstill : 22|1@1+ (1,0) [0|3] "" XXX
  SG_ Throttle_Cruise : 0|12@1+ (1,0) [0|255] "" XXX
- SG_ Counter_1 : 46|3@0+ (1,0) [0|15] "" XXX
- SG_ Wheel_stop : 22|1@0+ (1,0) [0|3] "" XXX
- SG_ CloseDistance : 24|8@1+ (1,0) [0|255] "" XXX
+ SG_ Cruise_Activatedish : 16|1@1+ (1,0) [0|7] "" XXX
  SG_ Unknown : 18|1@1+ (1,0) [0|7] "" XXX
- SG_ Button_Speed_Down : 49|1@1+ (1,0) [0|3] "" XXX
- SG_ Button_Resume : 50|1@1+ (1,0) [0|15] "" XXX
- SG_ NEW_SIGNAL_3_Blank : 15|4@0+ (1,0) [0|15] "" XXX
- SG_ NEW_SIGNAL_2_Blank : 48|1@1+ (1,0) [0|3] "" XXX
- SG_ Checksum : 56|8@1+ (1,0) [0|15] "" XXX
+ SG_ Brake_On : 20|1@1+ (1,0) [0|7] "" XXX
+ SG_ DistanceSwap : 21|1@1+ (1,0) [0|3] "" XXX
+ SG_ Counter : 44|3@1+ (1,0) [0|15] "" XXX
+ SG_ SET_0_3 : 47|1@0+ (1,0) [0|3] "" XXX
  SG_ NEW_SIGNAL_9 : 17|1@1+ (1,0) [0|3] "" XXX
- SG_ Cruise_Activatedish : 16|1@1+ (1,0) [0|7] "" XXX
- SG_ NEW_SIGNAL_6_Blank : 23|1@0+ (1,0) [0|7] "" XXX
- SG_ DistanceSwap : 21|1@0+ (1,0) [0|3] "" XXX
- SG_ Brake_On : 20|1@0+ (1,0) [0|7] "" XXX
- SG_ ES_Error : 42|1@1+ (1,0) [0|3] "" XXX
+ SG_ NEW_SIGNAL_1 : 19|1@1+ (1,0) [0|1] "" XXX
+ SG_ SET_0_4 : 43|1@0+ (1,0) [0|3] "" XXX
+ SG_ SET_0_1 : 32|9@1+ (1,0) [0|255] "" XXX
+ SG_ SET_1 : 23|1@1+ (1,0) [0|7] "" XXX
+ SG_ SET_0_2 : 55|5@0+ (1,0) [0|31] "" XXX
```

Comment:
can we keep the endianess consistent?

Reply:
Yes, my bad, missed it in cabana
I'll fix all messages and start a new PR

Reply:
addressed here: https://github.com/commaai/opendbc/pull/196